docs/api/doc-blocks/doc-block-story.mdx

```diff
@@ -76,6 +76,10 @@ Because all stories render simultaneously in docs entries, play functions can pe
 
 However, if you know your play function is “safe” to run in docs, you can use this prop to run it automatically.
 
+<Callout variant="info">
+  When the play functions uses `mount`, the story won't be rendered in docs. You must set autoplay to true for such stories if you want to use them in docs.
```

Comment:
```suggestion
  When a story's play function uses `mount`, the story won't be rendered in docs. You must set autoplay to true for such stories if you want to use them in docs.
```

Reply:
Also you should link to the `autoplay` docs here

Reply:
This is under the `autoplay` API reference heading.